graylog2-web-interface/src/util/conditional/filterMenuItems.js

```diff
@@ -0,0 +1,24 @@
+// @flow strict
+
+import isCloud from './isCloud';
+
+type MenuItem = { path: string };
+
+function filterMenuItems(
+  menuItems: Array<MenuItem>,
+  toFilter: Array<string>,
```

Comment:
Could we make it clearer that the array should contain the `path`s to filter? I think renaming the parameter to `pathsToFilter` would be enough.

Reply:
But wouldn't that imply that the array just contains paths, and not the whole menu items? I'd change it to `Array<string>` and just ask for the paths, makes it more reusable too. What do you think?

Reply:
I think making both arguments take the same data structure makes it clearer, so I like changing them to `Array<string>` and passing paths in this particular case.

Reply:
Will do that, thanks!